lightkurve/lightcurve.py

```diff
@@ -605,11 +605,11 @@ def fold(self, period, t0=None, transit_midpoint=None):
                                 label=self.label,
                                 meta=self.meta)
 
-    def normalize(self, unit='unscaled'):
+    def normalize(self, unit='unscaled', method='median'):
```

Comment:
Should we be able to pass a method here as well? e.g.
`lc.normalize(method=np.mean)`
`lc.normalize(method=astropy.stats.sigma_clipped_stats)`

Reply:
I understand this a bit of a fancy request, so I'm cool if we don't actually do it.

I think I would rather rely on `astropy`'s `sigma_clipped_stats` method for normalization, because then we could pass kwargs like `sigma` and `niters` in order to sigma clip and `mask` in order to mask...@barentsen what do you think?